ampd/src/event_processor.rs

```diff
@@ -137,6 +138,70 @@ fn is_avalanche_evm_handler(handler_label: &str) -> bool {
     evm_handlers.iter().any(|handler| handler_label == *handler)
 }
 
+fn show_protobuf_structure(bytes: &[u8]) -> Result<String, Box<dyn std::error::Error>> {
```

Comment:
could you add a TODO to indicate that this code should be removed at some point?